packages/multi-trait-rubric/src/scale.jsx

```diff
@@ -7,68 +7,79 @@ import Trait from './trait';
 
 const styles = () => ({
   table: {
-    border: '1px solid lightgrey',
     borderSpacing: 0,
     marginBottom: '16px',
+    borderRadius: '4px',
+    color: '#050F2D',
```

Comment:
can you check in with @elodszopos about color theming support? - most colors we use now should be themable

Reply:
sure

Reply:
I talked with Elod and added a new color in pie-lib/render-ui/colors, this is the PR for that: https://github.com/pie-framework/pie-lib/pull/259. It should be merged first, since I used the new color in this PR.